command/harness/dlite/init.go

```diff
@@ -24,10 +22,9 @@ type VMInitTask struct {
 }
 
 type VMInitRequest struct {
-	SetupVMRequest      harness.SetupVMRequest      `json:"setup_vm_request"`
-	Services            []*harness.ExecuteVMRequest `json:"services"`
-	Distributed         bool                        `json:"distributed,omitempty"`
-	GitspaceAgentConfig types.GitspaceAgentConfig   `json:"gitspace_agent_config"`
```

Comment:
Why are we removing GitspaceAgentConfig

Reply:
This is not needed (and was not used). There's GitspaceAgentConfig in SetupVMRequest which we are using.